scripts/embargo_registrations.py

```diff
@@ -0,0 +1,52 @@
+"""Run nightly, this script will activate any pending embargoes that have
+elapsed the pending approval time and make public and registrations whose
+embargo end dates have been passed.
+"""
+
+import datetime
+import logging
+import sys
+
+from modularodm import Q
+
+from website import models, settings
+from website.app import init_app
+from scripts import utils as scripts_utils
+
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def main(dry_run=True):
+    pending_embargoes = models.Embargo.find(Q('state', 'eq', 'unapproved'))
+    for embargo in pending_embargoes:
+        if should_be_embaroged(embargo):
+            embargo.state = 'active'
```

Comment:
Could use logging here about what is being changed. You can include the GUID of the embargo, etc.